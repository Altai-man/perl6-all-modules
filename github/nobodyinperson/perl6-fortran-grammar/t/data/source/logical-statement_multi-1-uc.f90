.TRUE. .AND. .FALSE. .OR. .NOT. .TRUE.
