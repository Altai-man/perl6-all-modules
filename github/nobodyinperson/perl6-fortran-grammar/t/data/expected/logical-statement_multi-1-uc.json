{
  "logical-operator": [
    ".AND.",
    ".OR."
  ],
  "logical-value-returning-code": [
    {
      "boolean": ".TRUE. "
    },
    {
      "boolean": ".FALSE. "
    },
    {
      "boolean": {
        "logical-prefix-operator": ".NOT."
      }
    }
  ]
}
